### ISSUE TITLE
auto-delete files in saveactions plugin temporary folder

### ISSUE THREAD

**Issue by abyss-soft** [geany/geany#3296]
I would like files to be auto-deleted when they are already closed in the editor. So far I have written this solution ([link](https://github.com/abyss-soft/clear_temp_geany)), but maybe there is a better option?

**Comment by elextr** [comment]
Couple of notes:

1. current git geany does not store the session files in geany.conf any more
2. won't work with projects

**Comment by abyss-soft** [comment]
Just tested my solution on version 1.38:
1. removes temporary project files as needed
2. this version stores session files in geany.conf

If there is any other solution, I will be glad to hear it.

**Comment by elextr** [comment]
> Just tested my solution on version 1.38: ... this version stores session files in geany.conf

Yes, as I said the change is new to the current git version and will be in the next release when that happens.

What I meant about project files is that they contain a list of open files in the project, but your script only examines lists in geany.conf, so it will remove autosave backups that are "open" in project files.

**Comment by abyss-soft** [comment]
Thanks, now everything is clear. Tell me, where will the sessions of open files be stored in the new version?
Maybe then in the new implementation you can immediately implement this functionality? It's just that the disk is not clogged with junk files.

**Comment by elextr** [comment]
IIUC session.conf if no project is open, or the project file when a project is open.

**Comment by abyss-soft** [comment]
Thank you. So for now I will use my solution, and after the release I will remake it with a new one.
